pkg/monitor/api/types.go

```diff
@@ -551,3 +552,11 @@ func (m PolicyMatchType) String() string {
 	}
 	return "unknown"
 }
+
+// Entity prints the ID in a human readable string
+func Entity(id uint32) string {
```

Comment:
I think `Entity` as a naming here crosses two separate concepts and I don't think it's a good idea to mix them up. The input is a numeric Identity, which represents exactly one possible security context of the peer in the connection. An Entity on the other hand is a policy-level selector which covers multiple individual Identities (for example, `cluster` entity covers various different identities that are all conceptually part of the 'cluster').

I wonder whether we could just convert the uint32 from monitor side into a native security identity type instead and get the string conversion from there? I'm looking at this which seems to be equivalent to this function:

https://github.com/cilium/cilium/blob/91748c7e19b4afd3a7d30087b7ba5f69da25452f/pkg/identity/numericidentity.go#L364-L370